packages/tfjs-node-helpers/src/testing/metrics/pr-auc.ts

```diff
@@ -0,0 +1,40 @@
+import { binarize } from '../../utils/binarize';
+import { Metric } from '../metric';
+import { MetricCalculator, MetricCalculatorData } from '../metric-calculator';
+import { PrecisionMetricCalculator } from './precision';
+import { RecallMetricCalculator } from './recall';
+
+export class PRAucMetricCalculator extends MetricCalculator {
+  constructor(private thresholdNumber: number = 21) {
```

Comment:
@VladimirGlinskii Let's rename this to `numberOfThresholds`. Also, why is it `21` by default?

Reply:
21 thresholds = [0, 0.05, 0.10, 0.15, ..., 0.90, 0.95, 1], or 20 steps of length 0.05 starting from 0

Reply:
@VladimirGlinskii Got it, and thank you for cleaning this up! It looks much better now.